index.html

```diff
@@ -1430,14 +1430,14 @@ <h2>
       </h2>
 
       <p>
-Authentication is the mechanism by which the <a>DID controller(s)</a> can
-cryptographically prove that they are associated with that DID.
-See Section <a href="#binding-of-identity"></a>. Note
-that Authentication is separate from Authorization because the
-<a>DID controller(s)</a> may wish to enable others to update their DID Document
-(for example, to assist with key recovery as discussed in Section
+Authentication is the mechanism by which <a>DID controllers</a> can
+cryptographically prove that they are associated with a <a>DID</a>. For more
+information, see Section <a href="#binding-of-identity"></a>. Note that
+Authentication is separate from Authorization because <a>DID controllers</a>
```

Comment:
It's deliberately DID controller(s) to indicate there could be one or more for any given DID I believe. (Wait for one of the editors to confirm before reverting all of these, just in case!)
```suggestion
Authentication is separate from Authorization because <a>DID controller(s)</a>
```

Reply:
Some others (like the one in line 1440, below) should have the parentheses, but I think this one should be left as plural without parentheses.

Reply:
It's not necessary to use parentheses to indicate one or more. "controllers" by itself is sufficient. But if you prefer, I will change it back.

Reply:
As suggested, left as "controllers".